### ISSUE TITLE
Using i5 8400 with Gigabyte b360mds3h for high Sierra 

### ISSUE THREAD

**Issue by Amarj2007** [saikatm/gigabyte-b360m-d3h-hackintosh-efi#4]
What changes I need to do for high Sierra in your efi folder

**Comment by saikatm** [comment]
You have select different SMBIOS depending on your CPU.

**Comment by Amarj2007** [comment]
I have selected 18,2 in SMBIOS as I m using i5 8400, and changed serial number also .. is any other things need to be changed in high sierra.As you are using Mojave

**Comment by saikatm** [comment]
yes, that will work I think. let me know if you face any issue. mail:

**Comment by Amarj2007** [comment]
Your mail id plz

**Comment by Amarj2007** [comment]
My System Hardware Details -
Mobo- Gigabyte B360M DS3H
Cpu- i5 8400
Graphics- 1050ti
hackintosh - 10.13.6

I have used your clover , and changes SMBIOS , everything working except Hardware acceleration .

I am attaching my EFI , please help me to enable Hardware Acceleration.

**Comment by Amarj2007** [comment]
[EFI.zip](https://github.com/saikatm/gigabyte-b360m-d3h-hackintosh-efi/files/3247103/EFI.zip)

**Comment by saikatm** [comment]
Hi, you need to install nvdia web driver to enable hardware acceleration of the gpu. I will look in to the efi folder once i go home

**Comment by Amarj2007** [comment]
I have installed driver...

**Comment by saikatm** [comment]
What is you BIOS setting? is igpu is enabled BIOS?

**Comment by saikatm** [comment]
use latest Lilu+Whatevegreen and recommended platform-id (0x3E9B0007).

**Comment by saikatm** [comment]
Also you might try SMBIOS iMac18,3

**Comment by saikatm** [comment]
![Screenshot 2019-06-03 19 38 57](https://user-images.githubusercontent.com/9909763/58808256-504c1f00-8637-11e9-99fd-08d7eb162ef9.png)
Also, try enabling/disabling inject intel under clover settings

**Comment by Amarj2007** [comment]
igpu is enabled in bios , will send you my bios settings.

**Comment by saikatm** [comment]
Can you show me the **About This Mac** section of your Hackintosh?

**Comment by Amarj2007** [comment]
Can you help me making Hackintosh Mojave for FCPX and audio editing.
Specs -
i5 8400
Gigabyte B360M Ds3H
Ram 16gb ddr4 2400
Rx 570 .

I have purchased RX 570 so that Mojave can work.

Please guide me what I will have to do for perfect working hackintosh.

**Comment by saikatm** [comment]
Hi, @Amarj2007 I have built my Hackintosh mainly for using video editing(FCPX).
It is been more than 6months and I have never faced any issue my Hackintosh is super stable.
Also good that you bought RX570 it will work natively on macOS.

I am glad to help you. please email me here: itzskt@gmail.com
